ci/helm/values/values-nuxeo.yaml.gotmpl

```diff
@@ -19,10 +19,10 @@ service:
     dev.nuxeo.com/resource: service
 mongodb:
   enabled: true
-  host: mongodb.{{ .Namespace }}.svc.cluster.local
+  host: mongodb{{ eq .Environment.Name "functional-tests-2023" | ternary 6 4 }}.{{ .Namespace }}.svc.cluster.local
```

Comment:
We talked about overriding the host in mongo values but I didn't find the right value yet. However. This condition is also done just above for ES / OS just under (line 25) so we only have one value to change.

Reply:
If it ends to be not possible, it would be better to have a complete if/else block on enabled values, it will help searching for mongodb6 in the repository (we can't have this reference right now) and help future environment if we can still deploy with the same chart.

Reply:
I'm gonna try with mongo's `.Values.service.nameOverride` I have high hopes about it 💪 
https://github.com/bitnami/charts/blob/443e30cd350100a00578e43f136ff96b3ee1899c/bitnami/mongodb/templates/_helpers.tpl#L27

Reply:
It was without service: https://github.com/bitnami/charts/blob/main/bitnami/common/templates/_names.tpl#L26-L27